test/SourceKit/CompileNotifications/diagnostics.swift

```diff
@@ -19,9 +19,9 @@
 // PARSE-WITH-SOURCELOCATION: key.notification: source.notification.compile-did-finish
 // PARSE-WITH-SOURCELOCATION-NEXT: key.diagnostics: [
 // PARSE-WITH-SOURCELOCATION-NEXT:   {
-// PARSE-WITH-SOURCELOCATION-NEXT:     key.line: 2000
+// PARSE-WITH-SOURCELOCATION-NEXT:     key.line: 2
 // PARSE-WITH-SOURCELOCATION-NEXT:     key.column: 6
-// PARSE-WITH-SOURCELOCATION-NEXT:     key.filepath: "custom.swuft"
+// PARSE-WITH-SOURCELOCATION-NEXT:     key.filepath: "{{.*}}parse-error-with-sourceLocation.swift"
```

Comment:
I assume this needs to be reverted to match reverting the editor diagnostic consumer change?

Reply:
Indeed 😳